Mathlib/LinearAlgebra/TensorProduct/RightExactness.lean

```diff
@@ -416,6 +417,58 @@ theorem TensorProduct.map_ker :
   rw [range_eq_top.mpr (rTensor_surjective M' hg), Submodule.map_top]
   rw [Exact.linearMap_ker_eq (lTensor_exact P hfg' hg')]
 
+variable (M)
+
+/-- Left tensoring a module with a quotient of the ring is the same as
+quotienting that module by the corresponding submodule. -/
+noncomputable def lTensor_ring_mod_ideal_equiv_mod_ideal_smul (I : Ideal R) :
```

Comment:
I think we usually call it `quot` or `quotient` rather than `mod`. And this is a def so it should be in snake case. I'd call this `quotTensorEquivQuotSMul` (and the other one `tensorQuotEquivQuotSMul`) myself, but any name should work.